web/app/view/ReportConfigController.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Anton Tananaev (anton.tananaev@gmail.com)
+ * Copyright 2016 Andrey Kunitsyn (abyss@fox5.ru)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Ext.define('Traccar.view.ReportConfigController', {
+    extend: 'Ext.app.ViewController',
+    alias: 'controller.reportConfigDialog',
+
+    init: function () {
+        var store = this.lookupReference('eventTypeField').getStore();
+        if (store.getCount() === 0) {
+            store.add({
+                type: Traccar.store.ReportEventTypes.allEvents,
```

Comment:
OK. I'll try to sum up:
1. We have to import stores and models in `Application.js` and should not import it again if access via `Ext.getStore()` or objects config (like grid or combobox).
We get error or warning if missed something.
2. We have to import all things that access directly except `Traccar.Style` and `Traccar.AttributeFormatter` because we usually just get 'undefined' with no error.
3. Chained import view is enough. I mean import view that imports controller.

Am I right?

One question:
If we create model or store via full name and `Ext.create()` should we import it again if it is already imported in `Application.js` ?